release/RELEASE_GUIDELINES.md

```diff
@@ -0,0 +1,31 @@
+# Release Guidelines for the Onfido JS SDK
+We follow semver for versioning. Given a version number MAJOR.MINOR.PATCH, increment the:
+
+- MAJOR version when you make incompatible API changes
+- MINOR version when you add functionality in a backwards-compatible manner
+- PATCH version when you make backwards-compatible bug fixes
+
+# Prerequisite
+In order to perform the release you need to:
+1. Download the Web SDK release configuration secret file and store it as `release/releaseSecret.js`
+  - Edit secret file where necessary
+2. Make sure your laptop is set up to be able to login to S3
+3. Make sure that your git configuration allows you to use git commands without inputting password or passphrase
+
+# Usage
+Run `VERSION=x.x.x node release/release.js`
```

Comment:
Do you think we need any instructions about the actions we will have to do when running the script?

Reply:
All the instructions should be provided by the release script. Should... 😛